---
layout: about
title: About
permalink: /
subtitle: <a href='https://www.hgsc.bcm.edu/' target="_blank">Human Genome Sequencing Center Baylor College of Medicine</a>, <a href='https://csweb.rice.edu/' target="_blank">Department of Computer Science, Rice University.</a>,
profile:
  align: right
  image: lab-retreat.jpg
  image_circular: false # crops the image to make it circular
  more_info: >
    <p>One Baylor Plaza</p>
    <p>Houston, TX 77030</p>

news: true # includes a list of news items
selected_papers: false # includes a list of papers marked as "selected={true}"
social: true # includes social icons at the bottom of the page
---

Our research focuses on the detection and understanding of genomic Structural Variation (SV) origins and mechanisms, to advance comparative genomics, biomedical research and ultimately, clinical care. SVs are an understudied form of genomic variation that includes deletions, duplications, insertions, inversions, and translocations. They represent an important class of variation; their overall impact on the Human genome is greater than the impact of single nucleotide variations. The core strength of the Sedlazeck lab is to develop new methods and algorithms to obtain a deeper understanding of the underlying molecular biology. We are doing this with a focus on large scale genomics (hundreds of thousands of samples), by applying novel technologies (e.g. long reads such as PacBio or Oxford Nanopore). Utilizing this we uncovered novel complexity across human evolution, diseases (e.g. neurological, mendelian, cardiovascular), cancer (e.g. breast or viral associated), but also non-human based. The latter either to improve our understanding of human variations (e.g. rhesus macaque) or to improve certain phenotypes (e.g. tomato, yeast)

Going forward, the Sedlazeck lab utilizes novel biotechnology and develops algorithms to improve our understand of human diseases with a special focus on adult diseases. To achieve this, we have experience in phasing, methylation calling and variant calling in complex regions of the human genome. We also leverage both mapping and assembly approaches to obtain a comprehensive insight into analyzed samples. In addition to comprehensive genomics, we apply these approaches at scale to improve heritability and association of certain diseases. We achieve it by applying scalable algorithms, cloud infrastructure and utilizing machine learning techniques together with high filtering and quality control standards. As such, our lab contributes to Topmed, CCDG, ADSP and other consortia to push innovation across genomics and genetics, addressing questions of evolution, diseases and development.

<!-- Write your biography here. Tell the world about yourself. Link to your favorite [subreddit](http://reddit.com). You can put a picture in, too. The code is already in, just name your picture `prof_pic.jpg` and put it in the `img/` folder.

Put your address / P.O. box / other info right below your picture. You can also disable any of these elements by editing `profile` property of the YAML header of your `_pages/about.md`. Edit `_bibliography/papers.bib` and Jekyll will render your [publications page](/al-folio/publications/) automatically.

Link to your social media connections, too. This theme is set up to use [Font Awesome icons](https://fontawesome.com/) and [Academicons](https://jpswalsh.github.io/academicons/), like the ones below. Add your Facebook, Twitter, LinkedIn, Google Scholar, or just disable all of them. -->
