---
layout: page
permalink: /software/
title: Softwares
description: 
nav: true
nav_order: 4
---

| Tool | Description |
| -------|---------|
| [BioBricks](https://github.com/fritzsedlazeck/BioBricks) | Software to simulate recombination to assess ancestry blocks |
| [FALCON](https://github.com/PacificBiosciences/FALCON) | Phased Diploid Genome Assembly with PacBio sequencing reads |
| [GenomeScope](http://genomescope.org) | Estimate genomic properties from unassembled sequencing reads |
| [LRSim](https://github.com/aquaskyline/LRSIM) | Linked Read Simulator |
| [MuMmer SGE](https://github.com/fritzsedlazeck/sge_mummer) | Parallization script for MUMMer. |
| [NGM](https://github.com/Cibiv/NextGenMap) | Short read mapper especially suitable for high SNP rates |
| [NGM-LR](https://github.com/philres/ngmlr) | Fast and Accurate Mapping of Long Reads |
| [Paragraph](https://github.com/Illumina/paragraph) | Genome graph based SV genotyping |
| [PhaseME](https://github.com/smajidian/phaseme) | Phasing QC and utilization of linkage data |
| [Princess](https://github.com/MeHelmy/princess) | Software for haplotype resolved detection of SNV, SV and methylation |
| [Read2Tree](https://github.com/dvdylus/read2tree) | Software for mapping reads directly onto orthology groups |
| [Sniffles](https://github.com/fritzsedlazeck/Sniffles) | Detection of Structural Variations from Long Reads |
| [SURVIVOR](https://github.com/fritzsedlazeck/SURVIVOR) | Detection and Analysis of Structural Variations |
| [SVCollector](https://github.com/fritzsedlazeck/SVCollector)| Method to optimally select samples for validation and resequencing based on a multi sample VCF. |
| [SVHound](https://github.com/lfpaulin/SVhound) | Statistical software to predict SV in a cohort that have not been found yet. |
| [Teaser](http://teaser.cibiv.univie.ac.at/) | Fast personalized benchmarks and optimization for NGS read mapping |
| [Vulcan](https://gitlab.com/treangenlab/vulcan) | Utilizing two scoring functions to improve long read mapping |