 <!-- About Section  -->
    <section id="publications">
        <div class="container">
            <div class="row">
                <div class="col-lg-12 text-center">
                    <h2>Publications</h2>
                    <hr class="star-primary">
                </div>
            </div>
            <div class="row">

                <table class="intro">

                    <tr><td colspan=3><h3>Preprints</h3><hr></td></tr>





                    <tr>
                        <td valign=top align=right>P10.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1101/2022.06.08.495395">A pan-genome approach to decipher variants in the highly complex tandem repeat of LPA</a><br>
                            C. Chin, S. Behera, GA. Metcalf, RA. Gibbs, E. Boerwinkle, <b>F.J. Sedlazeck</b>(2022) <i>bioRxiv</i> doi: https://doi.org/10.1101/2022.06.08.495395
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1101/2022.06.08.495395"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>P9.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1101/2022.08.03.502734"> Characterizing the genetic polymorphisms in 370 challenging medically relevant genes using long-read sequencing data from 41 human individuals among 19 global populations</a><br>
                            C. Ji, J. Gong, <b>F.J. Sedlazeck*</b>, S. Fan*(2022) <i>bioRxiv</i> doi: https://doi.org/10.1101/2022.08.03.502734
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1101/2022.08.03.502734"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>


                    <tr>
                        <td valign=top align=right>P8.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1101/2022.06.27.497715"> Frequent spontaneous structural rearrangements promote transgenerational genome diversification in Brassica napus</a><br>
                            M. Orantes-Bonilla, M. Makhoul, H. Lee, H. Singh Chawla, P. Vollrath, A. Langstroff, <b>F.J. Sedlazeck</b>, J. Zou, RJ. Snowdon(2022) <i>bioRxiv</i> doi: https://doi.org/10.1101/2022.06.27.497715
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1101/2022.06.27.497715"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>P7.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1101/2022.08.05.502980">Multiscale Analysis of Pangenome Enables Improved Representation of Genomic Diversity For Repetitive And Clinically Relevant Genes</a><br>
                            C. Chin, S. Behera, A.Khalak, <b>F.J. Sedlazeck</b>, J. Wagner, JM. Zook(2022) <i>bioRxiv</i> doi: https://doi.org/10.1101/2022.08.05.502980
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1101/2022.08.05.502980"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>



                    <tr>
                        <td valign=top align=right>P6.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1101/2022.04.27.489683">Improved sequence mapping using a complete reference genome and lift-over</a><br>
                            N.C. Chen, L.F. Paulin,  <b>F.J. Sedlazeck</b>, S. Koren, A.M. Phillippy, B. Langmead(2022) <i>bioRxiv</i> doi: https://doi.org/10.1101/2022.04.27.489683
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1101/2022.04.27.489683"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>


                    <tr>
                        <td valign=top align=right>P5.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1101/2022.04.18.488678">Read2Tree: scalable and accurate phylogenetic trees from raw reads</a><br>
                            D. Dylus, AM. Altenhoff,  S. Majidian, <b>F.J. Sedlazeck*</b> , C. Dessimoz*   (2022) <i>bioRxiv</i> doi: https://doi.org/10.1101/2022.04.18.488678
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1101/2022.04.18.488678"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>P4.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1101/2022.04.04.487055">Comprehensive Structural Variant Detection: From Mosaic to Population-Level</a><br>
                            M. Smolka,  L.F. Paulin,  C.M. Grochowski,  M. Mahmoud,  S. Behera, M. Gandhi,  K. Hong,  D. Pehlivan,  S.W. Scholz,  C.M.B. Carvalho,  C. Proukakis, <b>F.J. Sedlazeck</b> (2022) <i>bioRxiv</i> doi: https://doi.org/10.1101/2022.04.04.487055
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1101/2022.04.04.487055"><img width=50 height=70 src="img/paper/Sniffles.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>


                    <tr>
                        <td valign=top align=right>P3.</td>
                        <td valign=top>
                            <a href="https://www.biorxiv.org/content/10.1101/2021.04.26.441442v1">Multi-tissue integrative analysis of personal epigenomes</a><br>
                            J. Rozowsky, J. Drenkow, YT. Yang, G. Gursoy, T. Galeev, B. Borsari, C.B Epstein, K. Xiong, J. Xu, J. Gao, K. Yu, A. Berthel, Z. Chen, F. Navarro, J. Liu, MS. Sun, J. Wright, J. Chang, CJF. Cameron, N. Shoresh, E. Gaskell, J. Adrian, S. Aganezov, G. Balderrama-Gutierrez, S. Banskota, GB. Corona, S. Chee, SB. Chhetri, GCC. Martins, C. Danyko, CA. Davis, D. Farid, NP. Farrell, I. Gabdank, Y. Gofin, DU. Gorkin, M. Gu, V. Hecht, BC. Hitz, R. Issner, M. Kirsche, X. Kong, B.R Lam, S. Li, B. Li, T. Li, X. Li, KZ. Lin, R. Luo, M. Mackiewicz, J.E Moore, J. Mudge, N. Nelson, C. Nusbaum, I. Popov, HE. Pratt, Y. Qiu, S. Ramakrishnan, J. Raymond, L. Salichos, A. Scavelli, JM. Schreiber, <b>F.J. Sedlazeck</b>, LH. See, RM. Sherman, X. Shi, M. Shi, CA. Sloan, JS. Strattan, Z. Tan, FY. Tanaka, A.VJ. Wang, J. Werner, B. Williams, M. Xu, C. Yan, Lu Yu, C. Zaleski, J. Zhang, JM. Cherry, EM. Mendenhall, WS. Noble, Z. Weng, ME. Levine, A. Dobin, B. Wold, A. Mortazavi, B. Ren, J. Gillis, RM. Myers, MP. Snyder, J. Choudhary, A. Milosavljevic, MC. Schatz, R. Guigó, BE Bernstein, TR. Gingeras, M. Gerstein (2021) <i>bioRxiv</i> doi: https://doi.org/10.1101/2021.04.26.441442
                        </td>
                        <td valign=top> <a href="https://www.biorxiv.org/content/10.1101/2021.04.26.441442v1"><img width=50 height=70 src="img/paper/MultiTissue.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>


                    <tr>
                        <td valign=top align=right>P2.</td>
                        <td valign=top>
                            <a href="https://www.biorxiv.org/content/10.1101/2021.07.05.451071v1"> Construction of a new chromosome-scale, long-read reference genome assembly of the Syrian hamster, Mesocricetus auratus</a><br>
                            R.A. Harris, M. Raveendran, D.T. Lyfoung, <b>F.J. Sedlazeck</b>, M. Mahmoud, T.M. Prall, J.A. Karl, H. Doddapaneni, Q. Meng, Y. Han, D. Muzny, R.W. Wiseman, D.H. O’Connor, J. Rogers (2021) <i>bioRxiv</i> doi: https://doi.org/10.1101/2021.07.05.451071
                        </td>
                        <td valign=top> <a href="https://www.biorxiv.org/content/10.1101/2021.07.05.451071v1"><img width=50 height=70 src="img/paper/Hamster.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>P1.</td>
                        <td valign=top>
                            <a href="https://www.biorxiv.org/content/10.1101/2020.05.02.074096v1">Multiethnic catalog of structural variants and their translational impact for disease phenotypes across 19,652 genomes</a><br>
                            <b>F.J. Sedlazeck</b>, B. Yu, A.J. Mansfield, H. Chen, O. Krasheninina, A. Tin, Q. Qi, S. Zarate, J.L. Traynelis, V. Menon, BCM HGSC Sequencing Lab, J. Hu, H. Doddapaneni, G. A. Metcalf, J. Coresh, R. C. Kaplan, D.M. Muzny, G. Jun, R.A. Gibbs, W.J. Salerno, E. Boerwinkle (2021) <i>bioRxiv</i> doi: https://doi.org/10.1101/2020.05.02.074096
                        </td>
                        <td valign=top> <a href="https://www.biorxiv.org/content/10.1101/2020.05.02.074096v1"><img width=50 height=70 src="img/paper/CCDGSV.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr><td></td></tr>
                    <tr><td></td></tr>


                    <!--       2023        -->
                    <tr><td colspan=3><h3>2023</h3><hr></td></tr>

                    <tr>
                        <td valign=top align=right>98.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1186/s13059-023-02863-7">FixItFelix: improving genomic analysis by fixing reference errors.</a><br>
                            S. Behera, J. LeFaive, P. Orchard, M. Mahmoud, L. F. Paulin, J. Farek, D. C. Soto, S.C.J. Parker, A. V. Smith, M. Y. Dennis, J. M. Zook, <b>F.J. Sedlazeck</b>(2023) <i>Genome Biology</i> doi:https://doi.org/10.1186/s13059-023-02863-7
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1186/s13059-023-02863-7"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>97.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1158/2159-8290.CD-22-0900">Intratumoral heterogeneity and clonal evolution induced by HPV integration</a><br>
                            K. Akagi, DE. Symer, M. Mahmoud, B. Jiang, S. Goodwin, D. Wangsa, Z. Li, W. Xiao, JD. Dunn, R. McCombie, T. Ried, KR. Coombes,  <b>F.J. Sedlazeck</b>, ML. Gillison(2023) <i>Cancer Discovery</i> doi:https://doi.org/10.1158/2159-8290.CD-22-0900
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1158/2159-8290.CD-22-0900"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>96.</td>
                        <td valign=top>
                            <a href=" https://doi.org/10.1093/gigascience/giac125">xAtlas: Scalable small variant calling across heterogeneous next-generation sequencing experiments</a><br>
                            J. Farek, D. Hughes, W. Salerno, Y. Zhu, A. Pisupati, A. Mansfield, O. Krasheninina, A.C. English, G. Metcalf, E. Boerwinkle, D.M Muzny, R.A. Gibbs, Z. Khan, <b>F.J. Sedlazeck</b> (2023) <i>Gigascience</i> doi:https://doi.org/10.1093/gigascience/giac125
                        </td>
                        <td valign=top> <a href=" https://doi.org/10.1093/gigascience/giac125"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>95.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1186/s12859-022-05046-6">SVhound: Detection of future Structural Variation hotspots</a><br>
                            L.F. Paulin, M. Raveendran, R.A. Harris, J. Rogers, A.v. Haeseler, <b>F.J. Sedlazeck</b> (2023) <i>BCM Bioinformatics</i> doi:https://doi.org/10.1186/s12859-022-05046-6
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1186/s12859-022-05046-6"><img width=50 height=70 src="img/paper/SVHound.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>
                    <tr>
                        <td valign=top align=right>94.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1002/ana.26608">Genome-wide analysis of Structural Variants in Parkinson's Disease</a><br>
                            K.J. Billingsley, J. Ding, P.A. Jerez, A. Illarionova, K. Levine, F.P. Grenn, M.B. Makarious, A. Moore, D. Vitale, X. Reed, D. Hernandez, A. Torkamani, M. Ryten, J. Hardy, UK Brain Expression Consortium (UKBEC), R. Chia, S.W. Scholz, B.J. Traynor, C.L. Dalgard, D.J. Ehrlich, T. Tanaka, L. Ferrucci, T.G. Beach, G.E. Serrano, J.P. Quinn, V.J. Bubb, R.L. Collins, X. Zhao, M. Walker, E. Pierce-Hoffman, H. Brand, M.E. Talkowski, B. Casey, M.R. Cookson, A. Markham, M.A. Nalls, M. Mahmoud, <b>F.J. Sedlazeck</b> , C. Blauwendraat, J. R. Gibbs, A.B. Singleton (2023) <i>Annals of Neurology</i> doi:https://doi.org/10.1002/ana.26608
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1002/ana.26608"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>
                    <tr>
                        <td valign=top align=right>93.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1002/ajmg.a.63080">Genome-wide analysis of Structural Variants in Parkinson's Disease</a><br>
                            M. Dawood, G. Akay, T. Mitani, D. Marafi, J.M. Fatih, A. Gezdirici, H. Najmabadi, K. Kahrizi, J. Punetha, C.M. Grochowski, H. Du, A. Jolly, H. Li, Z. Coban‐Akdemir, <b>F.J. Sedlazeck</b>, J.V. Hunter, S.N. Jhangiani, D. Muzny, D. Pehlivan, J.E. Posey, C.MB. Carvalho, R.A. Gibbs, J.R. Lupski(2023) <i>American Journal of Medical Genetics </i> doi:https://doi.org/10.1002/ajmg.a.63080
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1002/ajmg.a.63080"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>





                    <!--       2022        -->

                    <tr><td colspan=3><h3>2022</h3><hr></td></tr>


                    <tr>
                        <td valign=top align=right>92.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1186/s13059-022-02840-6">Truvari: Refined Structural Variant Comparison Preserves Allelic Diversity</a><br>
                            A.C. English, V.K. Menon, R. Gibbs, G.A. Metcalf, <b>F.J. Sedlazeck</b> (2022) <i>Genome Biology</i> doi: https://doi.org/10.1186/s13059-022-02840-6
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1186/s13059-022-02840-6"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>


                    <tr>
                        <td valign=top align=right>91.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1002/humu.24461">Long read sequencing and expression studies of AHDC1 deletions in Xia‐Gibbs syndrome reveal a novel genetic regulatory mechanism</a><br>
                            V. Chander, M. Mahmoud, J. Hu, Z. Dardas, C. M. Grochowski, M. Dawood, M.M. Khayat, H. Li, S. Li, S. Jhangiani, V. Korchina, H. Shen, G. Weissenberger, Q. Meng, M. Gingras, D.M. Muzny, H. Doddapaneni, J.E. Posey, J.R. Lupski, A. Sabo, D.R. Murdock, <b>F.J. Sedlazeck</b>, RA. Gibbsn (2022) <i>Human Mutation </i> doi: https://doi.org/10.1002/humu.24461
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1002/humu.24461"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>



                    <tr>
                        <td valign=top align=right>90.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.3389/fpls.2022.1057953">Frequent spontaneous structural rearrangements promote rapid genome diversification in a Brassica napus F1 generation</a><br>
                            M. Orantes-Bonilla, M. Makhoul, H. Lee, H.S. Chawla, P. Vollrath, A. Langstroff, <b>F.J. Sedlazeck</b>, J. Zou, R.J. Snowdon (2022) <i>Frontiers in Plant Science </i> doi: https://doi.org/10.3389/fpls.2022.1057953
                        </td>
                        <td valign=top> <a href="https://doi.org/10.3389/fpls.2022.1057953"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>


                    <tr>
                        <td valign=top align=right>89.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.12688/f1000research.110194.1">The third international hackathon for applying insights into large-scale genomic composition to use cases in a wide range of organisms</a><br>
                            K. Walker, D. Kalra, R. Lowdon, G. Chen, D. Molik, D.C. Soto, F. Dabbaghie, A.A. Khleifat, M. Mahmoud, L.F. Paulin, M.S. Raza, S.P. Pfeifer, D.P. Agustinho, E. Aliyev, P. Avdeyev, E.R. Barrozo, S. Behera, K. Billingsley, L.C. Chong, D. Choubey, W.D. Coster, Y. Fu, A.R. Gener, T. Hefferon, D.M. Henke, W. Höps, A. Illarionova, M.D. Jochum, M. Jose, R.K. Kesharwani, S. Kolora, J. Kubica, P. Lakra, D. Lattimer, C. Liew, B. Lo, C. Lo, A. Lötter, S. Majidian, S.K. Mendem, R. Mondal, H. Ohmiya, N. Parvin, C. Peralta, C. Poon, R. Prabhakaran, M. Saitou, A. Sammi, P. Sanio, N. Sapoval, N. Syed, T. Treangen, G. Wang, T. Xu, J. Yang, S. Zhang, W. Zhou, <b>F.J. Sedlazeck</b>, B. Busby (2022) <i>F1000 </i> doi:https://doi.org/10.12688/f1000research.110194.1
                        </td>
                        <td valign=top> <a href="https://doi.org/10.12688/f1000research.110194.1"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>88.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1186/s13059-022-02735-6">Multiple genome alignment in the telomere-to-telomere assembly era</a><br>

                            Bryce Kille, Advait Balaji,  <b>F.J. Sedlazeck</b>, Michael Nute, Todd J Treangen (2022) <i>Genome biology </i> doi: https://doi.org/10.1186/s13059-022-02735-6
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1186/s13059-022-02735-6"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>


                    <tr>
                        <td valign=top align=right>87.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1186/s13073-022-01123-w">The multiple de novo copy number variant (MdnCNV) phenomenon presents with peri-zygotic DNA mutational signatures and multilocus pathogenic variation</a><br>

                            H. Du, A. Jolly, C.M. Grochowski, B. Yuan, M. Dawood, S.N. Jhangiani, H. Li, D. Muzny, J.M. Fatih, Z. Coban-Akdemir, M.E. Carlin, A.E. Scheuerle, K. Witzl, JE. Posey, M. Pendleton, E. Harrington, S. Juul, PJ. Hastings, W. Bi, RA. Gibbs,  <b>F.J. Sedlazeck</b>, JR. Lupski, C. Carvalho, P. Liu (2022) <i>Genome Medicine </i> doi: https://doi.org/10.1186/s13073-022-01123-w
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1186/s13073-022-01123-w"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>


                    <tr>
                        <td valign=top align=right>86.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1038/s42003-022-03610-7">Comprehensive short and long read sequencing analysis for the Gaucher and Parkinson’s disease-associated GBA gene</a><br>
                            M. Toffoli, X. Chen, <b>F.J. Sedlazeck</b>, C. Lee, S. Mullin, A. Higgins, S. Koletsi, ME. Garcia-Segura, E. Sammler, SW. Scholz, AHV. Schapira, MA. Eberle, C. Proukakis(2022) <i>Communications Biology volume </i> doi: https://doi.org/10.1038/s42003-022-03610-7
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1038/s42003-022-03610-7"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>



                    <tr>
                        <td valign=top align=right>85.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1093/gigascience/giac039">Construction of a new chromosome-scale, long-read reference genome assembly for the Syrian hamster, Mesocricetus auratus</a><br>
                            RA. Harris, M. Raveendran, DT. Lyfoung, <b>F.J. Sedlazeck</b>, M. Mahmoud, TM. Prall, JA. Karl, H. Doddapaneni, Q. Meng, Y. Han, D. Muzny, RW. Wiseman, DH. O'Connor, J. Rogers(2022) <i>Gigascience</i> doi: https://doi.org/10.1093/gigascience/giac039
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1093/gigascience/giac039"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>


                    <tr>
                        <td valign=top align=right>84.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1016/j.ajhg.2022.04.006">TOP-LD: A tool to explore linkage disequilibrium with TOPMed whole-genome sequence data</a><br>
                            L. Huang, J.D. Rosen, Q. Sun, J. Chen, M.M. Wheeler, Y. Zhou, Y. Min, C. Kooperberg, M.P. Conomos, A.M. Stilp, S.S. Rich, J.I. Rotter, A. Manichaikul, R.J.F. Loos, E.E. Kenny, T.W. Blackwell, A.V. Smith, G. Jun, <b>F.J. Sedlazeck</b>, G. Metcalf, E. Boerwinkle, LM. Raffield, A.P. Reiner, P.L. Auer, Y. Li (2022) <i>The American Journal of Human Genetics</i> doi: https://doi.org/10.1016/j.ajhg.2022.04.006
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1016/j.ajhg.2022.04.006"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>83.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1016/j.xgen.2022.100129">precisionFDA Truth Challenge V2: Calling variants from short- and long-reads in difficult-to-map regions</a><br>
                            N.D. Olson, J. Wagner, J. McDaniel, S.H. Stephens, S.T. Westreich, A.G. Prasanna, E. Johanson, E. Boja, E.J. Maier, O. Serang,  D.Jáspez, J.M. Lorenzo-Salazar, A. Muñoz-Barrera, L.A. Rubio-Rodríguez, C. Flores, K. Kyriakidis, A. Malousi, K. Shafin, T. Pesout, M. Jain, B. Paten, P. Chang, A. Kolesnikov, M. Nattestad, G. Baid, S. Goel, H. Yang, A. Carroll, R. Eveleigh, M. Bourgey, G. Bourque, G. Li, M. ChouXian, L. Tang, D. YuanPing, S. Zhang, J. Morata,  R. Tonda,  G. Parra, J. Trotta, C. Brueffer, S. Demirkaya-Budak, D. Kabakci-Zorlu, D. Turgut, Ö. Kalay, G. Budak, K. Narcı, E. Arslan, R. Brown, I.J. Johnson, A. Dolgoborodov, V. Semenyuk, A. Jain, H.S. Tetikol, V. Jain, M. Ruehle, B. Lajoie, C. Roddey, S. Catreux, R. Mehio, M.U. Ahsan, Q. Liu, K. Wang, S.M.E. Sahraeian,  L.T. Fang, M. Mohiyuddin, C. Hung, C. Jain, H. Feng, Z. Li, L. Chen, <b>F.J. Sedlazeck</b>,  J.M. Zook (2022) <i>Cell Genomics</i> doi: https://doi.org/10.1016/j.xgen.2022.100129
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1016/j.xgen.2022.100129"><img width=50 height=70 src="img/paper/FDA2.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>82.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1016/j.xgen.2022.100128">Benchmarking challenging small variants with linked and long reads</a><br>
                            J. Wagner, N.D. Olson, L. Harris, Z. Khan, J. Farek, M. Mahmoud, A. Stankovic, V. Kovacevic, A.M. Wenger, W.J. Rowell, C. Xiao, B. Yoo, N. Miller, J.A. Rosenfeld, B. Ni, S. Zarate, M. Kirsche, S. Aganezov, M.C. Schatz, G. Narzisi, M. Byrska-Bishop, W. Clarke, U.S. Evani, C. Markello, K. Shafin, X. Zhou, A. Sidow, V. Bansal, P. Ebert, T. Marschall, P. Lansdorp, V. Hanlon, C. Mattsson, A.M. Barrio, I.T. Fiddes, A. Fungtammasan, C. Chin, <b>F.J. Sedlazeck</b>, A. Carroll, M. Salit, J.M. Zook, Genome in a Bottle Consortium(2022) <i>Cell Genomics</i> doi: https://doi.org/10.1016/j.xgen.2022.100128
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1016/j.xgen.2022.100128"><img width=50 height=70 src="img/paper/GIABSNV.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>81.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1038/s41592-022-01423-4"> Searching thousands of genomes to classify somatic and novel structural variants using STIX</a><br>
                            M. Chowdhury, B.S. Pedersen, <b>F.J. Sedlazeck</b>,  A.R. Quinlan, R.M. Layer (2022) <i>Nature Methods</i> doi: https://doi.org/10.1038/s41592-022-01423-4
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1038/s41592-022-01423-4"><img width=50 height=70 src="img/paper/STIX.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>80.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1126/science.abj6987">The complete sequence of a human genome</a><br>
                            S. Nurk, S. Koren, A. Rhie, M. Rautiainen, AV. Bzikadze, A. Mikheenko, MR. Vollger, N. Altemose, L. Uralsky, A. Gershman, S. Aganezov, SJ. Hoyt, M. Diekhans, GA. Logsdon, M. Alonge, SE. Antonarakis, M. Borchers, GG. Bouffard, SY. Brooks, GV. Caldas, H. Cheng, C. Chin, W. Chow, LG. de Lima, PC. Dishuck, R. Durbin, T. Dvorkina, IT. Fiddes, G. Formenti, RS. Fulton, A. Fungtammasan, E. Garrison, PGS. Grady, TA. Graves-Lindsay, IM. Hall, NF. Hansen, GA. Hartley, M. Haukness, K. Howe, MW. Hunkapiller, C. Jain, M. Jain, ED. Jarvis, P. Kerpedjiev, M. Kirsche, M. Kolmogorov, J. Korlach, M. Kremitzki, H. Li, VV. Maduro, T. Marschall, AM. McCartney, J. McDaniel, DE. Miller, JC. Mullikin, EW. Myers, ND. Olson, B. Paten, P. Peluso, PA. Pevzner, D. Porubsky, T. Potapova, EI. Rogaev, JA. Rosenfeld, SL. Salzberg, VA. Schneider, <b>F.J. Sedlazeck</b>, K. Shafin, CJ. Shew, A. Shumate, Y. Sims, AFA. Smit, DC. Soto, I. Sović, JM. Storer, A.Streets, BA. Sullivan, F. Thibaud-Nissen, J. Torrance, J. Wagner, BP. Walenz, A. Wenger, JMD. Wood, C. Xiao, SM. Yan, AC. Young, S. Zarate, U.Surti, RC. McCoy, MY. Dennis, IA. Alexandrov, JL. Gerton, RJ. O’Neill, W. Timp, JM. Zook, MC. Schatz, EE. Eichler, KH. Miga, AM. Phillippy (2022) <i>Science</i> doi: https://doi.org/10.1126/science.abj6987
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1126/science.abj6987"><img width=50 height=70 src="img/paper/T2Tgenome.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>79.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1126/science.abl3533">A complete reference genome improves analysis of human genetic variation</a><br>
                            S. Aganezov, SM. Yan, DC. Soto, M. Kirsche, S. Zarate, P. Avdeyev, DJ. Taylor, K. Shafin, A. Shumate, C. Xiao, J. Wagner, J. McDaniel, ND. Olson, MEG. Sauria, MR. Vollger, A. Rhie, M. Meredith, S. Martin, J. Lee, S. Koren, J. Rosenfeld, B. Paten, R. Layer, C. Chin, <b>F.J. Sedlazeck</b>, NF. Hansen, DE. Miller, AM. Phillippy, KH. Miga, RC. McCoy*, MY. Dennis*, JM. Zook*, MC. Schatz* (2022) <i>Science</i> doi: https://doi.org/10.1126/science.abl3533
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1126/science.abl3533"><img width=50 height=70 src="img/paper/T2TVar.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>78.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1038/s41587-022-01221-5">Accelerated identification of disease-causing variants with ultra-rapid nanopore genome sequencing</a><br>
                            SD. Goenka,JE. Gorzynski, K. Shafin, DG. Fisk, T. Pesout, TD. Jensen,J. Monlong,  P. Chang, G. Baid, JA. Bernstein, JW. Christle, KP. Dalton, DR. Garalde, ME. Grove, J. Guillory, A. Kolesnikov, M. Nattestad, M.R.Z. Ruzhnikov, M. Samadi, A. Sethia, E. Spiteri, C.J. Wright, K. Xiong, T. Zhu, M. Jain, <b>F.J. Sedlazeck</b>, A. Carroll, B. Paten & E.A. Ashley (2022) <i>Nature Biotechnology</i> doi: https://doi.org/10.1038/s41587-022-01221-5
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1038/s41587-022-01221-5"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>


                    <tr>
                        <td valign=top align=right>77.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1038/s41467-022-28852-1">Rescuing Low Frequency Variants within Intra-Host Viral Populations directly from Oxford Nanopore sequencing data</a><br>
                            Y. Liu, J. Kearney, M. Mahmoud, B. Kille, <b>F.J. Sedlazeck</b>, T.J. Treangen (2022) <i>Nature Communications</i> doi: https://doi.org/10.1038/s41467-022-28852-1
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1038/s41467-022-28852-1"><img width=50 height=70 src="img/paper/cov_ont.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>76.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1186/s13059-022-02636-8">Towards accurate and reliable resolution of structural variants for clinical diagnosis</a><br>
                            Z. Liu,R. Roberts, TR. Mercer, J. Xu, <b>F.J. Sedlazeck*</b>, W. Tong* (2022) <i> Genome Biology</i> doi: https://doi.org/10.1186/s13059-022-02636-8
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1186/s13059-022-02636-8"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>

                        <td valign=top align=right>75.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1093/gigascience/giac010">Fully resolved assembly of Cryptosporidium parvum</a><br>
                            V.K. Menon, P.C. Okhuysen, C. Chappell, M. Mahmoud, Q. Meng, H. Doddapaneni, V. Vee, Y. Han, S. Salvi, S. Bhamidipati, K. Kottapalli, G. Weissenberger, H. Shen, M.C. Ross, K.L. Hoffman, S.J. Cregeen, D.M. Muzny, G.A. Metcalf, R.A. Gibbs, J.F. Petrosino, <b>F.J. Sedlazeck</b> (2022) <i>GigaScience</i> doi: https://doi.org/10.1093/gigascience/giac010
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1093/gigascience/giac010"><img width=50 height=70 src="img/paper/Crypto.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>74.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1016/j.gim.2021.12.005">Centers for Mendelian Genomics: A decade of facilitating gene discovery</a><br>
                            S.M. Baxter, J.E. Posey, N.J. Lake, N. Sobreira, J.X. Chong, S. Buyske, E.E. Blue, L.H. Chadwick, Z.H. Coban-Akdemir, K.F. Doheny, C.P. Davis, M. Lek,C. Wellington, S.N. Jhangiani, M. Gerstein, R.A. Gibbs, R.P. Lifton, D.G. MacArthur,T.C. Matise, J.R. Lupski, D. Valle, M. J.Bamshad, A. Hamosh, S. Mane, D.A.Nickerson, <b>Centers for Mendelian Genomics Consortium</b> ,H.L.Rehm, A. O’Donnell-Luria(2022) <i>GigaScience</i> doi: https://doi.org/10.1016/j.gim.2021.12.005
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1016/j.gim.2021.12.005"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>73.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1161/CIRCGEN.121.003591">Ultra-Rapid Nanopore Whole Genome Genetic Diagnosis of Dilated Cardiomyopathy in an Adolescent With Cardiogenic Shock</a><br>
                            JE. Gorzynski, K. Shafin,TD. Jensen, DG. Fisk,E. Spiteri, T. Pesout, G. Baid, JA. Bernstein, P. Chang, JW. Christle, K. Dunn, DR. Garalde, JW. Knowles, A. Kolesnikov, M. Ma, T.Moscarello, M. Nattestad, M. Perez, M. Samadi, C. Wright, CJ. Wusthoff, T. Zhu, M. Jain, <b>F.J. Sedlazeck</b>, A. Carroll, B. Paten, EA. Ashley (2022) <i> Circulation: Genomic and Precision Medicine</i> doi: https://doi.org/10.1161/CIRCGEN.121.003591
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1161/CIRCGEN.121.003591"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>72.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1038/s41587-021-01158-1">Curated variation benchmarks for challenging medically relevant autosomal genes</a><br>
                            J. Wagner, ND. Olson, L. Harris, J. McDaniel, H. Cheng, A. Fungtammasan, Y. Hwang, R. Gupta, AM. Wenger, WJ. Rowell, ZM. Khan, J. Farek, Y. Zhu, A. Pisupati, M. Mahmoud, C. Xiao, B. Yoo, SME. Sahraeian, DE. Miller, D. Jaspez, JM. Lorenzo-Salazar, A. Munoz-Barrera, LA. Rubio-Rodriguez, C. Flores, G. Narzisi, US. Evani, WE. Clarke, J. Lee, CE. Mason, SE. Lincoln, KH. Miga, MTW. Ebbert, A. Shumate, H. Li, C. Chin*, JM. Zook*, <b>F.J. Sedlazeck*</b> (2022) <i>Nature Biotechnology</i> doi: https://doi.org/10.1038/s41587-021-01158-1
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1038/s41587-021-01158-1"><img width=50 height=70 src="img/paper/GIABMed.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>71.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1056/NEJMc2112090">Ultrarapid Nanopore Genome Sequencing in a Critical Care Setting</a><br>
                            JE. Gorzynski, K. Shafin,TD. Jensen, DG. Fisk,E. Spiteri, T. Pesout, G. Baid, JA. Bernstein, P. Chang, JW. Christle, K. Dunn, DR. Garalde, JW. Knowles, A. Kolesnikov, M. Ma, T.Moscarello, M. Nattestad, M. Perez, M. Samadi, C. Wright, CJ. Wusthoff, T. Zhu, M. Jain, <b>F.J. Sedlazeck</b>, A. Carroll, B. Paten, EA. Ashley (2022) <i>New England Journal of Medicine</i> doi: https://doi.org/10.1056/NEJMc2112090
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1056/NEJMc2112090"><img width=50 height=70 src="img/paper/NEJM_ont.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>70.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1186/s13059-021-02569-8">Assessing reproducibility of inherited variants detected with short-read whole genome sequencing</a><br>
                            B. Pan, L. Ren, V. Onuchic, M. Guan, R. Kusko, S. Bruinsma, L. Trigg, A. Scherer, B. Ning, C. Zhang, C. Glidewell-Kenney, C. Xiao, E. Donaldson, <b>F.J. Sedlazeck</b>, G. Schroth, G. Yavas, H. Grunenwald, H. Chen, H. Meinholz, J. Meehan, J. Wang, J. Yang, J. Foox, J. Shang, K. Miclaus, L. Dong, L. Shi, M. Mohiyuddin, M. Pirooznia, P. Gong, R. Golshani, R. Wolfinger, S. Lababidi, S.M.E. Sahraeian, S. Sherry, T. Han, T. Chen, T. Shi, W. Hou, W. Ge, W. Zou, W. Guo, W. Bao, W. Xiao, X. Fan, Y. Gondo, Y. Yu, Y. Zhao, Z. Su, Z. Liu, W. Tong, W. Xiao, J.M. Zook, Y. Zheng, H. Hong(2022) <i>Genome Biology</i> doi: https://doi.org/10.1186/s13059-021-02569-8
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1186/s13059-021-02569-8"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>69.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1016/j.fsigen.2021.102629"> Accurate profiling of forensic autosomal STRs using the Oxford Nanopore Technologies MinION device</a><br>
                            C.L. Hall,  R.K. Kesharwani, N.R. Phillips, J.V. Planz, <b>F.J. Sedlazeck</b>,  R.R. Zascavage (2022) <i>Forensic Science International: Genetics</i> doi: https://doi.org/10.1016/j.fsigen.2021.102629
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1016/j.fsigen.2021.102629"><img width=50 height=70 src="img/paper/Forstr.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>


                    <tr><td colspan=3><h3>2021</h3><hr></td></tr>




                    <tr>
                        <td valign=top align=right>68.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1186/s13059-021-02558-x">Hidden biases in germline structural variant detection</a><br>
                            M.M. Khayat, S. Mohammad, E. Sahraeian, S. Zarate, A. Carroll, H. Hong, B. Pan, L. Shi, R.A. Gibbs, M. Mohiyuddin, Y. Zheng, <b>F.J. Sedlazeck</b>(2021) <i>Genome Biology</i> doi: https://doi.org/10.1186/s13059-021-02558-x
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1186/s13059-021-02558-x"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>67.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1093/nar/gkab812">High resolution copy number inference in cancer using short-molecule nanopore sequencing</a><br>
                            T. Baslan, S. Kovaka, <b>F.J. Sedlazeck</b>, Y. Zhang, R. Wappel, S.W. Lowe, S. Goodwin,  M.C. Schatz(2021) <i>Nucleic Acids Research</i> doi: https://doi.org/10.1093/nar/gkab812
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1093/nar/gkab812"><img width=50 height=70 src="img/paper/ONTCNV.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>66.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1186/s13059-021-02486-w">PRINCESS: comprehensive detection of haplotype resolved SNVs, SVs, and methylation</a><br>
                            M. Mahmoud, H. Doddapaneni, W. Timp, <b>F.J. Sedlazeck</b> (2021) <i>Genome Biology</i> doi: https://doi.org/10.1186/s13059-021-02486-w
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1186/s13059-021-02486-w"><img width=50 height=70 src="img/paper/princess.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>65.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1038/s41587-021-01049-5"> Performance assessment of DNA sequencing platforms in the ABRF Next-Generation Sequencing Study </a><br>
                            J. Foox , S. Tighe , C. Nicolet , J. Zook , M. Byrska-Bishop , W. Clarke , M. Khayat , M. Mahmoud , P. Laaguiby , Z. Herbert , D. Warner , G. Grills , J. Jen , S. Levy , J. Xiang , A. Alonso , X. Zhao , W. Zhang , F. Teng , Y. Zhao , H. Lu , G. Schroth , G. Narzisi , W. Farmerie , <b>F.J. Sedlazeck*</b>, D. Baldwin*, C. Mason* (2021) <i>Nature Biotechnology</i> doi: https://doi.org/10.1038/s41587-021-01049-5
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1038/s41587-021-01049-5"><img width=50 height=70 src="img/paper/comp.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>64.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1371/journal.pone.0244468">Oligonucleotide capture sequencing of the SARS-CoV-2 genome and subgenomic fragments from COVID-19 individuals</a><br>
                            H. Doddapaneni, S.J. Cregeen, R. Sucgang, Q. Meng, X. Qin, V. Avadhanula, H. Chao, V. Menon, E. Nicholson, D. Henke, F. Piedra, A. Rajan, Z. Momin, K. Kottapalli, K.L. Hoffman, <b>F.J. Sedlazeck</b>, G. Metcalf, P.A. Piedra, D.M. Muzny, J.F. Petrosino, R.A. Gibbs  (2021) <i>Plos One</i> doi: https://doi.org/10.1371/journal.pone.0244468
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1371/journal.pone.0244468"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>63.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1093/gigascience/giab063">Vulcan: Improved long-read mapping and structural variant calling via dual-mode alignment</a><br>
                            Y. Fu, M. Mahmoud, VV. Muraliraman, <b>F.J. Sedlazeck*</b>, TJ. Treangen* (2021) <i>Gigascience</i> doi: https://doi.org/10.1093/gigascience/giab063
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1093/gigascience/giab063"><img width=50 height=70 src="img/paper/Vulcan.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>62.</td>
                        <td valign=top>
                            <a href="https://www.nature.com/articles/s41576-021-00367-3">Towards population-scale long-read sequencing</a><br>
                            W.De Coster, M.H. Weissensteiner, <b>F.J. Sedlazeck</b> (2021) <i>Nature Reviews Genetics</i> doi: https://doi.org/10.1038/s41576-021-00367-3
                        </td>
                        <td valign=top> <a href="https://www.nature.com/articles/s41576-021-00367-3"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>61.</td>
                        <td valign=top>
                            <a href="https://movementdisorders.onlinelibrary.wiley.com/doi/10.1002/mds.28616">Intronic Haplotypes in the GBA Gene Do Not Predict Age at Diagnosis of Parkinson's Disease</a><br>
                            M. Toffoli, A. Higgins, C. Lee, S. Koletsi, X. Chen, M. Eberle, <b>F.J. Sedlazeck</b>, S. Mullin, C. Proukakis, A.H.V. Schapira (2021) <i>Movement Disorders</i> doi: https://doi.org/10.1002/mds.28616
                        </td>
                        <td valign=top> <a href="https://movementdisorders.onlinelibrary.wiley.com/doi/10.1002/mds.28616"><img width=50 height=70 src="img/paper/2021GBA.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>60.</td>
                        <td valign=top>
                            <a href="https://ashpublications.org/blood/article-abstract/doi/10.1182/blood.2021010858/475942/Investigation-of-product-derived-lymphoma?redirectedFrom=fulltext">Investigation of product derived lymphoma following infusion of piggyBac modified CD19 chimeric antigen receptor T-cells</a><br>

                            K.P. Micklethwaite, K. Gowrishankar, B.S. Gloss, Z. Li, J.A. Street, L. Moezzi, M.A. Mach, G. Sutrave, L.E. Clancy, D.C. Bishop, R.H.Y. Louie, C. Cai, J. Foox, M. Mackay, <b> F.J. Sedlazeck</b>, P. Blombery, C.E. Mason, F. Luciani, D.J. Gottlieb, E. Blyth (2021) <i>Blood</i> doi: https://doi.org/10.1182/blood.2021010858
                        </td>

                        <td valign=top> <a href="https://ashpublications.org/blood/article-abstract/doi/10.1182/blood.2021010858/475942/Investigation-of-product-derived-lymphoma?redirectedFrom=fulltext"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>59.</td>
                        <td valign=top>
                            <a href="https://f1000research.com/articles/10-246">An international virtual hackathon to build tools for the analysis of structural variants within species ranging from coronaviruses to vertebrates</a><br>

                            AM. Mc Cartney, M. Mahmoud, M. Jochum, DP. Agustinho, B.Zorman, AA. Khleifat, F. Dabbaghie, RK. Kesharwani, M. Smolka, M. Dawood, D. Albin, E. Aliyev, H. Almabrazi, A. Arslan, A. Balaji, S. Behera, K. Billingsley, DL. Cameron, J. Daw, ET. Dawson, W. De Coster, H. Du, C. Dunn, R. Esteban, A. Jolly, D. Kalra, C. Liao, Y. Liu, T. Lu, JM. Havrilla, MM. Khayat, M. Marin, J. Monlong, S. Price, A.R. Gener, J. Ren, S. Sagayaradj, N. Sapoval, C. Sinner, DC. Soto, A. Soylev, A. Subramaniyan, N. Syed, N. Tadimeti, P. Tater, P. Vats, J. Vaughn, K. Walker, G. Wang, Q. Zeng, S. Zhang, T. Zhao, B. Kille, E. Biederstedt, M. Chaisson, A. English, Z. Kronenberg, TJ. Treangen, T. Hefferon, C. Chin, B. Busby,<b> <b> F.J. Sedlazeck</b></b>(2021) <i>F1000Research</i> doi: https://doi.org/10.12688/f1000research.51477.1
                        </td>

                        <td valign=top> <a href="https://f1000research.com/articles/10-246"><img width=50 height=70 src="img/paper/Hack2020.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>58.</td>
                        <td valign=top>
                            <a href="https://genome.cshlp.org/content/31/5/910">Optimized sample selection for cost-efficient long-read population sequencing</a><br>

                            T.R. Ranallo-Benavidez, Z. Lemmon, S. Soyk, S. Aganezov, WJ. Salerno, RC. McCoy, ZB. Lippman, MC. Schatz,<b> <b> F.J. Sedlazeck</b></b>(2021) <i>Genome Research</i> doi: https://doi.org/10.1101/gr.264879.120
                        </td>
                        <td valign=top> <a href="https://genome.cshlp.org/content/31/5/910"><img width=50 height=70 src="img/paper/SVColl.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>57.</td>
                        <td valign=top>
                            <a href="https://www.sciencedirect.com/science/article/pii/S1096719221004716?via%3Dihub">Long-read sequencing for diagnosis in the Undiagnosed Diseases Network</a><br>

                            D. Murdock, J. Rosenfeld, F. Xia, L. Burrage, M. Mahmoud, <b> F.J. Sedlazeck</b>, J. Nguyen, M. Guzman, J. Xu, S. Wu, P. Rosillo, C. Martinez J. Campbell, S. Nicholas, E. Hanson, C. Poli, T. Vogel, B. Lee (2021) <i>Molecular Genetics and Metabolism</i> doi: https://doi.org/10.1016/S1096-7192(21)00471-6
                        </td>

                        <td valign=top> <a href="https://www.sciencedirect.com/science/article/pii/S1096719221004716?via%3Dihub"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>56.</td>
                        <td valign=top>
                            <a href="https://academic.oup.com/bioinformatics/article-abstract/37/14/2055/6184858?redirectedFrom=fulltext">muCNV: Genotyping Structural Variants for Population-level Sequencing</a><br>

                            G. Jun, <b> F.J. Sedlazeck</b>, Q. Zhu, A. English, G. Metcalf, H.M. Kang, C. Lee, R. Gibbs, E. Boerwinkle (2021) <i>Oxford Bioinformatics</i> doi: https://doi.org/10.1093/bioinformatics/btab199
                        </td>

                        <td valign=top> <a href="https://academic.oup.com/bioinformatics/article-abstract/37/14/2055/6184858?redirectedFrom=fulltext"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>55.</td>
                        <td valign=top>
                            <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7337385/">SARS-CoV-2 genomic diversity and the implications for qRT-PCR diagnostics and transmission</a><br>

                            N. Sapoval*, M. Mahmoud*, M.D. Jochum, Y. Liu, R.A.L. Elworth, Q. Wang, D. Albin, H. Ogilvie, M.D. Lee, S. Villapol, K.M. Hernandez, I.M. Berry, J. Foox, A. Beheshti, K. Ternus, K.M. Aagaard, D. Posada, C.E. Mason, <b> F.J. Sedlazeck</b>*, T.J. Treangen* (2021) <i>Genome Research</i> doi: https://doi.org/10.1101/gr.268961.120
                        </td>

                        <td valign=top> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7337385/"><img width=50 height=70 src="img/paper/covhid.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>54.</td>
                        <td valign=top>
                            <a href="https://www.nature.com/articles/s41467-021-21361-7">Shotgun Transcriptome and Isothermal Profiling of SARS-CoV-2 Infection Reveals Unique Host Responses, Viral Diversification, and Drug Interactions</a><br>

                            D. J. Butler, C. Mozsary, C. Meydan, D. Danko, J. Foox, J. Rosiene, A. Shaiber, E. Afshinnekoo, M. MacKay, <b> F.J. Sedlazeck</b>, N.A. Ivanov, M. Sierra, D. Pohle, M.Zietz, U. Gisladottir, V. Ramlall, C. D. Westover, K. Ryon, B. Young, C. Bhattacharya, P. Ruggiero, B.W. Langhorst, N. Tanner, J. Gawrys, D. Meleshko, D. Xu, P.A. D. Steel, A.J. Shemesh, J. Xiang, J. Thierry-Mieg, D. Thierry-Mieg, R.E. Schwartz, A. Iftner, D. Bezdan, J. Sipley, L. Cong, A. Craney, P. Velu, A. M. Melnick, I. Hajirasouliha, S. M. Horner, T. Iftner, M. Salvatore, M. Loda, L.F. Westblade, M. Cushing, S. Levy, S. Wu, N. Tatonetti, M. Imielinski, H. Rennert, C.E. Mason (2021) <i>Nature Communications</i> doi: https://doi.org/10.1038/s41467-021-21361-7
                        </td>

                        <td valign=top> <a href="https://www.nature.com/articles/s41467-021-21361-7"><img width=50 height=70 src="img/paper/CovMason.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>53.</td>
                        <td valign=top>
                            <a href="https://www.nature.com/articles/s41587-020-0711-0">Chromosome-scale, haplotype-resolved assembly of human genomes</a><br>

                            S. Garg, A. Fungtammasan, A. Carroll, M. Chou, A. Schmitt, X.Zhou, S. Mac, P. Peluso, E. Hatas, J. Ghurye, J. Maguire, M. Mahmoud, H. Cheng, D. Heller, J.M. Zook, T. Moemke, T. Marschall, <b> F.J. Sedlazeck</b>, J.Aach, Chen-S. Chin, G.M. Church, H. Li (2021) <i> Nature Biotechnology</i> doi: https://doi.org/10.1038/s41587-020-0711-0
                        </td>

                        <td valign=top> <a href="https://www.nature.com/articles/s41587-020-0711-0"><img width=50 height=70 src="img/paper/PhasedHum.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr><td></td></tr>
                    <tr><td></td></tr>

                    <tr><td colspan=3><h3>2020</h3><hr></td></tr>

                    <tr>
                        <td valign=top align=right>52.</td>
                        <td valign=top>
                            <a href="https://academic.oup.com/gigascience/article/9/12/giaa145/6042728">Parliament2: Fast Structural Variant Calling Using Optimized Combinations of Callers</a><br>

                            S. Zarate, A. Carroll, M. Mahmoud, O. Krashenina, G. Jun, W. Salerno, MC. Schatz, E. Boerwinkle, R. Gibbs,<b> <b> F.J. Sedlazeck</b></b>(2020) <i> GigaScience</i> doi: https://doi.org/10.1093/gigascience/giaa145
                        </td>

                        <td valign=top> <a href="https://academic.oup.com/gigascience/article/9/12/giaa145/6042728"><img width=50 height=70 src="img/paper/Par2.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>51.</td>
                        <td valign=top>
                            <a href="https://www.nature.com/articles/s41592-020-01000-7">Simultaneous profiling of chromatin accessibility and methylation on human cell lines with nanopore sequencing</a><br>

                            I. Lee, R. Razaghi, T. Gilpatrick, M. Molnar, N. Sadowski, J. T. Simpson, <b> F.J. Sedlazeck</b>, W. Timp (2020) <i> Nature Methods</i> doi: https://doi.org/10.1038/s41592-020-01000-7
                        </td>

                        <td valign=top> <a href="https://www.nature.com/articles/s41592-020-01000-7"><img width=50 height=70 src="img/paper/Nanome.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>50.</td>
                        <td valign=top>
                            <a href="https://genome.cshlp.org/content/early/2020/10/29/gr.262667.120">Complex mosaic structural variations in human fetal brains</a><br>

                            S. Sekar, L. Tomasini, C. Proukakis, T. Bae, L. Manlove, Y. Jang, S. Scuderi, B. Zhou, M. Kalyva, A. Amiri, J. Mariani, <b> F.J. Sedlazeck</b>, A.E. Urban, F.M. Vaccarino*, A. Abyzov*(2020) <i> Genome Research</i> doi: https://doi.org/10.1101/gr.262667.120
                        </td>

                        <td valign=top> <a href="https://genome.cshlp.org/content/early/2020/10/29/gr.262667.120"><img width=50 height=70 src="img/paper/brainSV.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>49.</td>
                        <td valign=top>
                            <a href="https://www.nature.com/articles/s41467-020-18564-9">A diploid assembly-based benchmark for variants in the major histocompatibility complex</a><br>

                            C. Chin, J. Wagner, Q. Zeng, E. Garrison, S. Garg, A. Fungtammasan, M. Rautiainen, S. Aganezov, M. Kirsche, S. Zarate, MC. Schatz, Chunlin X., WJ. Rowell, C. Markello, J. Farek, <b> F.J. Sedlazeck</b>, V. Bansal, B. Yoo, N. Miller, X. Zhou, A. Carroll, A.M. Barrio, M. Salit, T. Marschall, A.T. Dilthey, J.M. Zook(2020) <i> Nature Communications</i> doi: https://doi.org/10.1038/s41467-020-18564-9
                        </td>

                        <td valign=top> <a href="https://www.nature.com/articles/s41467-020-18564-9"><img width=50 height=70 src="img/paper/HLA.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>48.</td>
                        <td valign=top>
                            <a href="https://genome.cshlp.org/content/30/9/1258">Comprehensive analysis of structural variants in breast cancer genomes using single-molecule sequencing</a><br>

                            S. Aganezov, S. Goodwin, R.M. Sherman, <b> F.J. Sedlazeck</b>, G. Arun, S. Bhatia, I. Lee, M. Kirsche, R. Wappel, M. Kramer, K. Kostroff, D.L. Spector, W. Timp, W.R. McCombie, M.C. Schatz(2020) <i> Genome Research</i> doi: https://doi.org/10.1101/gr.260497.119
                        </td>

                        <td valign=top> <a href="https://genome.cshlp.org/content/30/9/1258"><img width=50 height=70 src="img/paper/CompSKRB3.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>47.</td>
                        <td valign=top>
                            <a href="https://academic.oup.com/gigascience/article/9/7/giaa078/5875849"> PhaseME: Automatic rapid assessment of phasing quality and phasing improvement</a><br>

                            S. Majidian,<b> <b> F.J. Sedlazeck</b></b>(2020) <i>  Giga Science</i> doi: https://doi.org/10.1093/gigascience/giaa078
                        </td>

                        <td valign=top> <a href="https://academic.oup.com/gigascience/article/9/7/giaa078/5875849"><img width=50 height=70 src="img/paper/PhaseMe.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>46.</td>
                        <td valign=top>
                            <a href="https://www.nature.com/articles/s41467-020-17195-4"> Discovery and population genomics of structural variation in a songbird genus</a><br>

                            M. H. Weissensteiner, I. Bunikis, A. Catalán, K.J. Francoijs, U. Knief, W. Heim, V. Peona, S. D. Pophaly, <b> F.J. Sedlazeck</b>, A. Suh, V. M. Warmuth, J. B.W. Wolf (2020) <i> Nature Communications</i> doi: https://doi.org/10.1038/s41467-020-17195-4
                        </td>

                        <td valign=top> <a href="https://www.nature.com/articles/s41467-020-17195-4"><img width=50 height=70 src="img/paper/SongBird.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>45.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1016/j.cell.2020.05.021"> Major impacts of widespread structural variation on gene expression and crop improvement in tomato</a><br>

                            M. Alonge, X. Wang, M. Benoit, S. Soyk, L. Pereira, L. Zhang, H. Suresh, S. Ramakrishnan, F. Maumus, D. Ciren, Y. Levy, T. H. Harel, G. Shalev-Schlosser, Z. Amsellem, H. Razifard, A.L. Caicedo, D. M. Tieman, H. Klee, M. Kirsche, S. Aganezov, T. R. Ranallo-Benavidez, Z. H. Lemmon, J. Kim, G. Robitaille, M. Kramer, S. Goodwin, W. R. McCombie, S. Hutton, J. Van Eck, J. Gillis, Y. Eshed, <b> F.J. Sedlazeck</b>, E. van der Knaap, M.C. Schatz*,Z. B. Lippman* (2020) <i> CELL</i> doi: https://doi.org/10.1016/j.cell.2020.05.021
                        </td>

                        <td valign=top> <a href="https://doi.org/10.1016/j.cell.2020.05.021"><img width=50 height=70 src="img/paper/TomCell.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>44.</td>
                        <td valign=top>
                            <a href="https://www.nature.com/articles/s41587-020-0538-8">A robust benchmark for germline structural variant detection</a><br>

                            M. Zook, N. F. Hansen, N. D. Olson, L. M. Chapman, J. C. Mullikin, C. Xiao, S. Sherry, S. Koren, A. M. Phillippy, P. C. Boutros, S. Mohammad, E. Sahraeian, V. Huang, A. Rouette, N. Alexander, C. E. Mason, I. Hajirasouliha, C. Ricketts, J. Lee, R. Tearle, I. T. Fiddes, A. M. Barrio, J. Wala, A. Carroll, N. Ghaffari, O. L. Rodriguez, A. Bashir, S. Jackman, J. J. Farrell, A. M. Wenger, C. Alkan, A. Soylev, M. C. Schatz, S. Garg, G. Church, T. Marschall, K. Chen, X. Fan, A. C. English, J. A. Rosenfeld, W. Zhou, R. E. Mills, J. M. Sage, J. R. Davis, M. D. Kaiser, J. S. Oliver, A. P. Catalano, M.JP Chaisson, N. Spies, <b> F.J. Sedlazeck</b>, M. Salit (2020) <i> Nature Biotechnology</i> doi: https://doi.org/10.1038/s41587-020-0538-8
                        </td>

                        <td valign=top> <a href="https://www.nature.com/articles/s41587-020-0538-8"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>43.</td>
                        <td valign=top>
                            <a href="https://www.nature.com/articles/s41576-021-00367-3"> Efficient de novo assembly of eleven human genomes using PromethION sequencing and a novel nanopore toolkit</a><br>

                            K. Shafin, T. Pesout, R. Lorig-Roach, M. Haukness, H. E. Olsen, C. Bosworth, J. Armstrong, K. Tigyi, N. Maurer, S. Koren, <b> F.J. Sedlazeck</b>, T. Marschall, S. Mayes, V. Costa, J. M. Zook, K. J. Liu, D. Kilburn, M. Sorensen, K. M. Munson, M. R. Vollger, E. E. Eichler, S. Salama, D. Haussler, R. E. Green, M. Akeson, A. Phillippy, K. H. Miga, P. Carnevali, M. Jain, B.Paten (2020) <i>  Nature Biotechnology</i> doi: https://doi.org/10.1038/s41587-020-0503-6
                        </td>

                        <td valign=top> <a href="https://www.nature.com/articles/s41576-021-00367-3"><img width=50 height=70 src="img/paper/GIABSV.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>42.</td>
                        <td valign=top>
                            <a href="https://www.nature.com/articles/s41587-020-0407-5"> Targeted Targeted nanopore sequencing with Cas9-guided adapter ligation</a><br>

                            T. Gilpatrick, I. Lee, J.E. Graham, E. Raimondeau, R. Bowen, A. Heron, <b> F.J. Sedlazeck</b>, W. Timp  (2020) <i>  Nature Biotechnology</i> doi: https://doi.org/10.1038/s41587-020-0407-5
                        </td>

                        <td valign=top> <a href="https://www.nature.com/articles/s41587-020-0407-5"><img width=50 height=70 src="img/paper/ONTCas9.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>41.</td>
                        <td valign=top>
                            <a href="https://pubmed.ncbi.nlm.nih.gov/32007927/">Potential applications of nanopore sequencing for forensic analysis</a><br>

                            CL. Hall, R.R. Zascavage, <b> F.J. Sedlazeck</b>, J.V. Planz  (2020) <i>  Forensic Sci Rev</i> doi: https://doi.org/10.1038/s41587-020-0407-5
                        </td>

                        <td valign=top> <a href="https://pubmed.ncbi.nlm.nih.gov/32007927/"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr><td></td></tr>
                    <tr><td></td></tr>

                    <tr><td colspan=3><h3>2019</h3><hr></td></tr>

                    <tr>
                        <td valign=top align=right>40.</td>
                        <td valign=top>
                            <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1909-7">Paragraph: A graph-based structural variant genotyper for short-read sequence data</a><br>

                            S. Chen, P. Krusche, E. Dolzhenko, R.M. Sherman, R. Petrovski, F. Schlesinger, M. Kirsche, D.R. Bentley, M. C. Schatz, <b> F.J. Sedlazeck</b>*, M.A. Eberle* (2019) <i>  Genome Biology</i> doi: https://doi.org/10.1186/s13059-019-1909-7
                        </td>

                        <td valign=top> <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1909-7"><img width=50 height=70 src="img/paper/Para.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>39.</td>
                        <td valign=top>
                            <a href="https://currentprotocols.onlinelibrary.wiley.com/doi/10.1002/cphg.94"> Approaches to Whole Mitochondrial Genome Sequencing on the Oxford Nanopore MinION</a><br>

                            R. R. Zascavage, C. L. Hall, K. Thorson, M. Mahmoud, <b> F.J. Sedlazeck</b>, J. V. Planz (2019) <i>  Current Protocols in Human Genetics</i> doi: https://doi.org/10.1002/cphg.94
                        </td>
                        <td valign=top> <a href="https://currentprotocols.onlinelibrary.wiley.com/doi/10.1002/cphg.94"><img width=50 height=70 src="img/paper/notAVL.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>38.</td>
                        <td valign=top>
                            <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1828-7"> Structural variant calling: the long and the short of it</a><br>

                            M. Mahmoud, N. Gobet, D. I. Cruz-DÃ¡valos, N. Mounier, C. Dessimoz*, <b> F.J. Sedlazeck</b>* (2019) <i> Genome Biology</i> doi: https://doi.org/10.1186/s13059-019-1828-7
                        </td>

                        <td valign=top> <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1828-7"><img width=50 height=70 src="img/paper/SVRev.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>37.</td>
                        <td valign=top>
                            <a href="https://www.cell.com/ajhg/fulltext/S0002-9297(19)30388-X">A Genocentric Approach to Discovery of Mendelian Disorders</a><br>

                            A. W. Hansen, M. Murugan, H. Li, M. M. Khayat, L. Wang, J. Rosenfeld, B. K. Andrews, S. N. Jhangiani, Z. H. Coban Akdemir, <b> F.J. Sedlazeck</b>, A. E. Ashley-Koch, P. Liu, D. M. Muzny, E. E. Davis, N. Katsanis, A. Sabo, J. E. Posey, Y. Yang, M. F. Wangler, C. M. Eng, V. Reid Sutton, J. R. Lupski, E. Boerwinkle, R. A. Gibbs (2019) <i> The American Journal of Human Genetics</i> doi: https://doi.org/10.1016/j.ajhg.2019.09.027
                        </td>

                        <td valign=top> <a href="https://www.cell.com/ajhg/fulltext/S0002-9297(19)30388-X"><img width=50 height=70 src="img/paper/DataLake.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>36.</td>
                        <td valign=top>
                            <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1829-6"> RaGOO: fast and accurate reference-guided scaffolding of draft genomes</a><br>

                            M. Alonge, S. Soyk, S. Ramakrishnan, X. Wang, S. Goodwin, <b> F.J. Sedlazeck</b>, Z. B. Lippman, M. C. Schatz (2019) <i> Genome Biology</i> doi: https://doi.org/10.1186/s13059-019-1829-6
                        </td>

                        <td valign=top> <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1829-6"><img width=50 height=70 src="img/paper/Ragoo.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>35.</td>
                        <td valign=top>
                            <a href="https://f1000research.com/articles/8-1751"> A strategy for building and using a human reference pangenome</a><br>

                            B. Llamas*, G. Narzisi*, V. Schneider*, P.A. Audano, E. Bieder- stedt, L. Blauvelt, P. Bradbury, X. Chang, C. Chin, A. Fungtammasan, W.E. Clarke, A. Cleary, J.Ebler, J. Eizenga, J. A. Sibbesen, C.J. Markello, E. Garrison, S. Garg, G. Hickey, G. R. Lazo, M. F. Lin, M. Mahmoud, T. Marschall, I. Minkin, J. Monlong, R. L. Musunuri, S. Sagayaradj, A.M. Novak, M.Rautiainen, A. Regier <b> F.J. Sedlazeck</b>, J. Siren, Y. Souilmi J. Wagner, T. Wrightsman, T. T. Yokoyama, Q. Zeng, J.M. Zook, B. Paten, B. Busby(2019) <i> F1000</i> doi: https://doi.org/10.12688/f1000research.19630.1
                        </td>

                        <td valign=top> <a href="https://f1000research.com/articles/8-1751"><img width=50 height=70 src="img/paper/Hack2019.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>34.</td>
                        <td valign=top>
                            <a href="https://academic.oup.com/gigascience/article/8/9/giz110/5565134"> Evaluation of computational genotyping of Structural Variations for clinical diagnoses</a><br>

                            V. Chander, R.A. Gibbs,<b> <b> F.J. Sedlazeck</b></b>(2019) <i> GigaScience</i> doi: https://doi.org/10.1093/gigascience/giz110
                        </td>

                        <td valign=top> <a href="https://academic.oup.com/gigascience/article/8/9/giz110/5565134"><img width=50 height=70 src="img/paper/SVGTCom.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>33.</td>
                        <td valign=top>
                            <a href="https://www.nature.com/articles/s41587-019-0217-9">Accurate circular consensus long-read sequencing improves variant detection and assembly of a human genome</a><br>

                            A.M. Wenger, P. Peluso, W.J. Rowell, P. Chang, R.J. Hall, G.T. Concepcion, J. Ebler, A. Fungtammasan, A. Kolesnikov, N.D. Olson, A. Toepfer, C. Chin, M. Alonge, M. Mahmoud, Y. Qian, A.M. Phillippy, M.C. Schatz, G. Myers, M.A. DePristo, J. Ruan, T. Marschall, <b> F.J. Sedlazeck</b>, J.M. Zook, H. Li, S. Koren, A. Carroll, D.R. Rank, M.W. Hunkapiller (2019) <i> Nature Biotechnology</i> doi: https://doi.org/10.1038/s41587-019-0217-9
                        </td>

                        <td valign=top> <a href="https://www.nature.com/articles/s41587-019-0217-9"><img width=50 height=70 src="img/paper/Hifi.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>


                    <tr>
                        <td valign=top align=right>32.</td>
                        <td valign=top>
                            <a href="https://academic.oup.com/mbe/article/36/9/1975/5492083">Ancestral admixture is the main determinant of global biodiversity in fission yeast</a><br>

                            S. Tusso, B.P.S. Nieuwenhuis, <b> F.J. Sedlazeck</b>, J.W. Davey, D.C. Jeffares, J.B.W. Wolf (2019) <i>Molecular Biology and Evolution</i> doi: https://doi.org/10.1093/molbev/msz126
                        </td>
                        <td valign=top> <a href="https://academic.oup.com/mbe/article/36/9/1975/5492083"><img width=50 height=70 src="img/paper/YestAnce.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>31.</td>
                        <td valign=top>
                            <a href="https://www.nature.com/articles/s41477-019-0422-z"> Duplication of a domestication locus neutralized a cryptic variant that caused a breeding barrier in tomato</a><br>

                            S. Soyk, Z.H. Lemmon, <b> F.J. Sedlazeck</b>, J.M. Jimenez-Gomez, M. Alonge, S.F. Hutton, J. Van Eck, M.C. Schatz, Z.B. Lippman (2019) <i>Nature Plants</i> doi: https://doi.org/10.1038/s41477-019-0422-z
                        </td>
                        <td valign=top> <a href="https://www.nature.com/articles/s41477-019-0422-z"><img width=50 height=70 src="img/paper/TomDup.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>30.</td>
                        <td valign=top>
                            <a href="https://www.nature.com/articles/s41436-019-0475-4"> Atlas- CNV: a validated approach to call Single-Exon CNVs in the eMERGESeq gene panel</a><br>

                            T. Chiang, X. Liu, T. Wu, J. Hu, <b> F.J. Sedlazeck</b>, S. White, D. Schaid, M. Andrade, G.P. Jarvik, D. Crosslin, I. Stanaway, D.S. Carrell, J.J. Connolly, H. Hakonarson, E.E. Groopman, A.G. Gharavi, A. Fedotov, W. Bi, M. S Leduc, D.R. Murdock, Y. Jiang, L. Meng, C.M. Eng, S. Wen, Y. Yang, D.M. Muzny, E. Boerwinkle, W. Salerno, E. Venner, R.A. Gibbs (2019) <i>Genetics in Medicine</i> doi: https://doi.org/10.1038/s41436-019-0475-4
                        </td>
                        <td valign=top> <a href="https://www.nature.com/articles/s41436-019-0475-4"><img width=50 height=70 src="img/paper/AtlasCNV.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>29.</td>
                        <td valign=top>
                            <a href="https://www.nature.com/articles/s41467-019-09025-z"> A multi-task convolutional deep neural network for variant calling in single molecule sequencing</a><br>

                            R. Luo, <b> F.J. Sedlazeck</b>, TW. Lam, M.C. Schatz (2019) <i>Nature Communication</i> doi: https://doi.org/10.1038/s41467-019-09025-z
                        </td>
                        <td valign=top> <a href="https://www.nature.com/articles/s41467-019-09025-z"><img width=50 height=70 src="img/paper/Clair.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>28.</td>
                        <td valign=top>
                            <a href="https://www.sciencedirect.com/science/article/pii/S0092867419301084">Megabase Length Hypermutation Accompanies Human Structural Variation at 17p11.2</a><br>

                            C.R. Beck, C.M.B. CarvalhoZ.C. Akdemir, <b> F.J. Sedlazeck</b>, X.Song, Q. Meng, J. Hu, H. Doddapaneni, Z. Chong, E.S. Chen, P.C. Thornton, P. Liu, B. Yuan, M. Withers, S.N. Jhangiani, D. Kalra, K. Walker, A.C. English, Y. Han, K. Chen, D.M. Muzny, G. Ira, C.A. Shaw, R.A. Gibbs, P.J. Hastings, J.R. Lupski (2019) <i>CELL</i> doi: https://doi.org/10.1016/j.cell.2019.01.045
                        </td>
                        <td valign=top> <a href="https://www.sciencedirect.com/science/article/pii/S0092867419301084"><img width=50 height=70 src="img/paper/CELLCNV.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>27.</td>
                        <td valign=top>
                            <a href="https://elifesciences.org/articles/41556">Combined transcriptome and proteome profiling reveals specific molecular brain signatures for sex, maturation and circalunar clock phase</a><br>

                            S. Schenk, S. Bannister, <b> F.J. Sedlazeck</b>, D. Anrather, B.Q. Minh, A. Bileck, M. Hartl, A. von Haeseler, C. Gerner, F. Raible, and K. Tessmar Raible (2019) <i>eLIFE</i> doi: https://doi.org/10.7554/eLife.41556
                        </td>
                        <td valign=top> <a href="https://elifesciences.org/articles/41556"><img width=50 height=70 src="img/paper/SeaWorm.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>26.</td>
                        <td valign=top>
                            <a href="https://pubmed.ncbi.nlm.nih.gov/30637984/">Evaluation of the detection of GBA missense mutations and other variants using the Oxford Nanopore MinION</a><br>

                            M. Leija Salazar, <b> F.J. Sedlazeck</b>, K. Mokretar, S. Mullin, M. Toffoli, M. Athanasopoulou, A. Donald, R. Sharma, D. Hughes, AH. Schapira, C. Proukakis  (2019) <i>Molecular Genetics & Genomic Medicine</i> doi: https://doi.org/10.1002/mgg3.564
                        </td>
                        <td valign=top> <a href="https://pubmed.ncbi.nlm.nih.gov/30637984/"><img width=50 height=70 src="img/paper/GBA.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr><td></td></tr>
                    <tr><td></td></tr>

                    <tr><td colspan=3><h3>2018</h3><hr></td></tr>

                    <tr>
                        <td valign=top align=right>25.</td>
                        <td valign=top>
                            <a href="https://onlinelibrary.wiley.com/doi/10.1111/mec.14969">Genome wide patterns of transposon proliferation in an evolutionary young hybrid fish</a><br>

                            S. Dennemoser, <b> F.J. Sedlazeck</b>, Michael C. Schatz, A.W. Nolte (2018) <i>Molecular Ecology</i> doi: https://doi.org/10.1111/mec.14969
                        </td>
                        <td valign=top> <a href="https://onlinelibrary.wiley.com/doi/10.1111/mec.14969"><img width=50 height=70 src="img/paper/FishLTR.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>24.</td>
                        <td valign=top>
                            <a href="https://genome.cshlp.org/content/28/8/1126">Complex rearrangements and oncogene amplifications revealed by long-read DNA and RNA sequencing of a breast cancer cell line</a><br>

                            M. Nattestad, K. Ng, S. Goodwin, T. Baslan, <b> F.J. Sedlazeck</b>, J. Gurtowski, E. Hutton, M. Alford, E. Tseng,J. Chin, T. Beck, Y. Sundaravadanam, M. Kramer, E. Antoniou, J. McPherson, J. Hicks, M.C. Schatz, W.R. McCombie (2018) <i>Genome Research</i> doi: https://doi.org/10.1101/gr.231100.117
                        </td>
                        <td valign=top> <a href="https://genome.cshlp.org/content/28/8/1126"><img width=50 height=70 src="img/paper/SKBR3.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>23.</td>
                        <td valign=top>
                            <a href="https://www.nature.com/articles/s41592-018-0001-7"> Accurate detection of complex structural variations using single-molecule sequencing</a><br>

                            <b> F.J. Sedlazeck</b>*, P. Rescheneder*, M. Smolka, H. Fang, M. Nattestad, A. Haeseler, M.C. Schatz (2018) <i> Nature Methods</i> doi: https://doi.org/10.1038/s41592-018-0001-7
                        </td>
                        <td valign=top> <a href="https://www.nature.com/articles/s41592-018-0001-7"><img width=50 height=70 src="img/paper/Sniffles.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>22.</td>
                        <td valign=top>
                            <a href="https://www.nature.com/articles/s41576-018-0003-4">Piercing the dark matter: bioinformatics of long-range sequencing and mapping</a><br>

                            <b> F.J. Sedlazeck</b>, L. Hayan, CA. Darby, M.C. Schatz (2018) <i>Nature Reviews Genetics</i> doi: https://doi.org/10.1038/s41576-018-0003-4
                        </td>
                        <td valign=top> <a href="https://www.nature.com/articles/s41576-018-0003-4"><img width=50 height=70 src="img/paper/PierceLR.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr><td></td></tr>
                    <tr><td></td></tr>

                    <tr><td colspan=3><h3>2017</h3><hr></td></tr>

                    <tr>
                        <td valign=top align=right>21.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1016/j.csbj.2017.10.002">LRSim: a Linked Reads Simulator generating insights for better genome partitioning</a><br>

                            R.Luo, <b> F.J. Sedlazeck</b>, C.A. Darby,S. Kelly, M.C. Schatz (2017) <i>Comput Struct Biotechnol.</i> doi: https://doi.org/10.1016/j.csbj.2017.10.002
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1016/j.csbj.2017.10.002"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>20.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.12688/f1000research.12516.1">Tools for annotation and comparison of structural variation</a><br>

                            <b> F.J. Sedlazeck</b>, A. Dhroso, D.L. Bodian, J. Paschall, F. Hermes, J. M. Zook(2017) <i>F1000Research</i> doi: https://doi.org/10.12688/f1000research.12516.1
                        </td>
                        <td valign=top> <a href="https://doi.org/10.12688/f1000research.12516.1"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>19.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.12688/f1000research.11254.1">DangerTrack: A scoring system to detect difficult-to-assess regions</a><br>

                            I. Dolgalev*, <b> F.J. Sedlazeck</b>*, Ben Busby(2017) <i>F1000Research</i> doi: https://doi.org/10.12688/f1000research.11254.1
                        </td>
                        <td valign=top> <a href="https://doi.org/10.12688/f1000research.11254.1"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>18.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1111/mec.14134">Copy number increases of transposable elements and protein coding genes in an invasive fish of hybrid origin</a><br>

                            S. Dennenmoser, <b> F.J. Sedlazeck</b>, E. Iwaszkiewicz, X. Li, J. Altmuller, A.W. Nolte(2017) <i>Molecular Ecology</i> doi: https://doi.org/10.1111/mec.14134
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1111/mec.14134"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>17.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1093/bioinformatics/btx153">GenomeScope: Fast genome-wide heterozygosity analysis from unassembled short reads</a><br>


                            G. Vurture*, <b> F.J. Sedlazeck</b>*, M. Nattestad, C. Underwood, H. Fang, J. Gurtowski, M.C. Schatz (2017) <i>Bioinformatics</i> doi: https://doi.org/10.1093/bioinformatics/btx153
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1093/bioinformatics/btx153"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>16.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1038/NCOMMS14061">Transient structural variations have strong effects on quantitative traits and reproductive isolation in fission yeast</a><br>


                            D.C. Jeffares, C. Jolly, M. Hoti, D. Speed, L. Shaw, C. Rallis, F. Balloux, C. Dessimoz*, J. Bahler*, <b> F.J. Sedlazeck</b>* (2017) <i>Nature Communication</i> doi: https://doi.org/10.1038/NCOMMS14061
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1038/NCOMMS14061"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr><td></td></tr>
                    <tr><td></td></tr>

                    <tr><td colspan=3><h3>2016</h3><hr></td></tr>

                    <tr>
                        <td valign=top align=right>15.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1038/nature20151">The genomic basis of circadian and circalunar timing adaptations in a midge</a><br>


                            T.S. Kaiser, B. Poehn, D. Szkiba, M. Preussner, <b> F.J. Sedlazeck</b>, A. Zrim, T. Neumann, L. Nguyen, A.J. Betancourt, T. Hummel, H. Vogel, S. Dorner, F. Heyd, A. von Haeseler, K. Tessmar-Raible (2016) <i>Nature</i> doi: https://doi.org/10.1038/nature20151
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1038/nature20151"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>14.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1038/nmeth.4035"> Phased diploid genome assembly with single-molecule real-time sequencing</a><br>


                            C.S. Chin, P. Peluso, <b> F.J. Sedlazeck</b>, M. Nattestad, G.T. Concepcion, A. Clum, C. Dunn, R.O. Malley, R. Figueroa-Balderas, A. Morales-Cruz, G.R. Cramer, M. Delledonne, C. Luo, J.R. Ecker, D. Cantu, D.R. Rank, M.C. Schatz (2016) <i>Nature Methods</i> doi: https://doi.org/10.1038/nmeth.4035
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1038/nmeth.4035"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>13.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1371/journal.pgen.1005954">Chromosomal- Level Assembly of the Asian Seabass Genome Using Long Sequence Reads and Multilayered Scaffolding</a><br>


                            S. Vij, H. Kuh, I. S. Kuznetsova, A. Komissarov, A. A.Yurchenko, P. V. Heusden, S. Singh, N. M. Thevasagayam, P.S.R. Sridatta, K. Purushothaman, J.M. Saju1, J. Jiang, S.K. Mbandi, M. Jonas, A.H.Y. Tong, S. Mwangi, D. Lau, S.Y. Ngoh, W.C. Liew, X. Shen, L.S. Hon, J.P. Drake, M. Boitano, R. Hall, J. Chin, R. Lachumanan, J. Korlach, V. Trifonov, M. Kabilov, A. Tupikin, D. Green, S. Moxon, T. Garvin, <b> F.J. Sedlazeck</b>, G.W. Vurture, G. Gopikrishna, V.K. K, T.H. Noble, V. Scaria, S. Sivasubbu, D.R. Jerry, S.J. O’Brien, M.C. Schatz, T. Dalmay, S. Turner, S. Lok, A. Christoffels, L. Orban (2016) <i>PLOS Genetics</i> doi: https://doi.org/10.1371/journal.pgen.1005954
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1371/journal.pgen.1005954"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr><td></td></tr>
                    <tr><td></td></tr>

                    <tr><td colspan=3><h3>2015</h3><hr></td></tr>

                    <tr>
                        <td valign=top align=right>12.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1038/ng.3435">The pineapple genome and the evolution of CAM photosynthesis</a><br>


                            R. Ming, R. VanBuren, C.M. Wai, H. Tang, M.C. Schatz, J.E. Bowers, E. Lyons, M. Wang, J. Chen, E. Biggers, J. Zhang, L. Huang, L. Zhang, W. Miao, J. Zhang, Z. Ye, C. Miao, Z. Lin, H. Wang, H. Zhou, W.C. Yim, H.D. Priest, C. Zheng, M. Woodhouse, P.P. Edger, R. Guyot, H. Guo, H. Guo, G. Zheng, R. Singh, A. Sharma, X. Min, Y. Zheng, H. Lee, J. Gurtowski, <b> F.J. Sedlazeck</b>, A. Harkess, M.R. McKain, Z. Liao, J. Fang, J. Liu, X. Zhang, Q. Zhang, W. Hu, Y. Qin, K. Wang, L. Chen, N. Shirley, Y. Lin, L. Liu, A.G. Hernandez, C.L. Wright, V. Bulone, G.A. Tuskan, K. Heath, F. Zee, P.H. Moore, R. Sunkar, J. H. Leebens-Mack, T. Mockler, J.L. Bennetzen, M. Freeling, D. Sankoff, A.H. Paterson, X. Zhu, X. Yang, J.A.C. Smith, J.C. Cushman, R.E. Paull, Q. Yu (2015) <i> Nature Genetics</i> doi: https://doi.org/10.1038/ng.3435
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1038/ng.3435"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>11.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1186/s13059-015-0803-1">Teaser: Individualized benchmarking and optimization of read mapping results for NGS data</a><br>


                            M. Smolka, P. Rescheneder, M.C. Schatz, A. von Haeseler,<b> <b> F.J. Sedlazeck</b></b>(2015) <i> Genome Biology</i> doi: https://doi.org/10.1186/s13059-015-0803-1
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1186/s13059-015-0803-1"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>10.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1371/journal.ppat.1005218">The Candida albicans Histone Acetyltransferase Hat1 Regulates Stress Resistance and Virulence via Distinct Chromatin Assembly Pathways</a><br>


                            M. Tscherner, F. Zwolanek, S. Jenull, <b> F.J. Sedlazeck</b>, I.E. Frohner, N. Chauhan, A. von Haeseler and K. Kuchler (2015) <i> PLOS Pathogen</i> doi: https://doi.org/10.1371/journal.ppat.1005218
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1371/journal.ppat.1005218"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>9.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1098/rspb.2015.0746">Ectodysplasin signaling genes and phenotypic evolution in sculpins (Cottus)</a><br>


                            J. Cheng, <b> F.J. Sedlazeck</b>, J. Altmuller, A.W. Nolte  (2015) <i> Proceedings B</i> doi: https://doi.org/10.1098/rspb.2015.0746
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1098/rspb.2015.0746"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr>
                        <td valign=top align=right>8.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1186/s12917-015-0378-9">Decreased expression of endogenous feline leukemia virus in cat lymphomas: a case control study</a><br>


                            M. Krunic, R. Ertle, B. Hagen, <b> F.J. Sedlazeck</b>, R. Hofmann- Lehmann, A. von Haeseler and D. Klein (2015) <i>BMC Veterinary Research</i> doi: https://doi.org/10.1186/s12917-015-0378-9
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1186/s12917-015-0378-9"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>
                    <tr><td></td></tr>

                    <tr><td></td></tr>
                    <tr><td></td></tr>

                    <tr><td colspan=3><h3>2014</h3><hr></td></tr>

                    <tr>
                        <td valign=top align=right>7.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1093/nar/gku844">ADAR2 induces reproducible changes in sequence and abundance of mature microRNAs in the mouse brain</a><br>


                            C. Vesely, S. Tauber, <b> F.J. Sedlazeck</b>, M. Tajaddod,A. von Haeseler, M.F. Jantsch (2014) <i>Nucleic Acids Research</i> doi: https://doi.org/10.1093/nar/gku844
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1093/nar/gku844"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>

                    <tr><td></td></tr>
                    <tr><td></td></tr>

                    <tr><td colspan=3><h3>2013</h3><hr></td></tr>

                    <tr>
                        <td valign=top align=right>6.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1093/bioinformatics/btt468"> NextGenMap: fast and accurate read mapping in highly polymorphic genomes</a><br>
                            <b> F.J. Sedlazeck</b>∗, P. Rescheneder∗, and A. von Haeseler (2013) <i>Bioinformatics</i> doi: https://doi.org/10.1093/bioinformatics/btt468
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1093/bioinformatics/btt468"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>

                    <tr>
                        <td valign=top align=right>5.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1038/nbt.2522"> Update on “Performance comparison of benchtop high-throughput sequencing platforms“</a><br>
                            S. Jünemann, <b> F.J. Sedlazeck</b>, K. Prior, A. Albersmeier, U. John, J. Kalinowski, A. Mellmann, A. Goesmann, A. von Haeseler, J. Stoye, and D. Harmsen (2013) <i>Nature Biotechnology</i> doi: https://doi.org/10.1038/nbt.2522
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1038/nbt.2522"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>

                    <tr>
                        <td valign=top align=right>4.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1016/j.ygeno.2012.12.001"> Benefit-of-doubt (BOD) scoring: a sequencing-based method for SNP candidate assessment from high to medium read number data sets</a><br>
                            <b> F.J. Sedlazeck</b>, P. Talloji, A. von Haeseler, and A. Bachmair(2013) <i>Genomics</i> doi: https://doi.org/10.1016/j.ygeno.2012.12.001
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1016/j.ygeno.2012.12.0016"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>

                    <tr><td></td></tr>
                    <tr><td></td></tr>

                    <tr><td colspan=3><h3>2012</h3><hr></td></tr>

                    <tr>
                        <td valign=top align=right>3.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1371/journal.pone.0041528"> Advanced Methylome Analysis after Bisulfite Deep Sequencing: an Example in Arabidopsis</a><br>
                            H.Q. Dinh, M. Dubin∗ <b> F.J. Sedlazeck</b>∗ N. Lettner, O. Mittelsten Scheid, and A. von Haeseler (2012) <i>PLoS ONE</i> doi: https://doi.org/10.1371/journal.pone.0041528
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1371/journal.pone.0041528"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>

                    <tr>
                        <td valign=top align=right>2.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.5220/0003775701950201"> MASon: Million Alignments In Seconds - A Platform Independent Pairwise Sequence Alignment Library for Next Generation Sequencing Data</a><br>
                            P. Rescheneder, A. von Haeseler, and<b> <b> F.J. Sedlazeck</b></b>(2012) <i> Proceedings of the International Conference on Bioinformatics Models,Methods and Algorithms (BIOINFORMATICS 2012) </i> doi: https://doi.org/10.5220/0003775701950201
                        </td>
                        <td valign=top> <a href="https://doi.org/10.5220/0003775701950201"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>

                    <tr>
                        <td valign=top align=right>1.</td>
                        <td valign=top>
                            <a href="https://doi.org/10.1101/gr.133025.111"> Adenosine deaminases that act on RNA induce reproducible changes in abundance and sequence of embryonic miRNAs</a><br>
                            C. Vesely∗, S. Tauber∗, <b> F.J. Sedlazeck</b>, A. von Haeseler, and M.F. Jantsch (2012) <i> Genome Research</i> doi: https://doi.org/10.1101/gr.133025.111
                        </td>
                        <td valign=top> <a href="https://doi.org/10.1101/gr.133025.111"><img width=50 height=70 src="img/paper/rev_2021.png"></a> </td>
                    </tr>
                </table>
                <br>
                <br>

                See more in <a href="https://scholar.google.com/citations?user=KNZTJ40AAAAJ&hl=en">Google Scholar</a>.

                <br>
                <br>
                <br>
                <br>



            </div>
        </div>
    </section>
